.github/workflows/continuous-integration.yml

```
@@ -8,9 +8,87 @@ on:
+          ini-values: |
+            zend.assertions=1
+            memory_limit=512M
+            opcache.jit=0
+            opcache.jit_buffer_size=0
+            opcache.enable=0
+            opcache.enable_cli=0
+
+      - name: Get composer cache directory
+        id: composer-cache
+        shell: bash
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)